systest/cluster/cluster.go

```diff
@@ -263,6 +273,17 @@ func (c *Cluster) AddSmeshers(cctx *testcontext.Context, n int) error {
 	return nil
 }
 
+// DeletePoet delete all poet servers.
```

Comment:
DeletePoet deletes first poet?

Reply:
the code actually delete ALL poets, as i intended to. updated the name